code/client/cl_main.c

```diff
@@ -3656,6 +3699,11 @@ void CL_Init( void ) {
 	cl_voipProtocol = Cvar_Get ("cl_voipProtocol", cl_voip->integer ? "opus" : "", CVAR_USERINFO | CVAR_ROM);
 #endif
 
+#ifdef USE_HTTP
+	if(!CL_HTTP_Init()) {
+		Com_Printf("WARNING: couldn't initialize HTTP\n");
```

Comment:
This could be more specific to avoid confusion such as "couldn't initialize HTTP download support".